### DISCUSSION TITLE
Logging Out in iOS Safari

### DISCUSSION THREAD

**Discussion by Heidieker** [usememos/memos#2870] [Q&A]
I am using multiple accounts. Since version 0.19.0, I have been unable to log out on mobile Safari for iOS. I am unable to click on the account name. Even switching to desktop view does not resolve the issue. Does anyone have any suggestions?

**Comment by boojack** [comment]
Thanks for the reporting, it will be fixed in `v0.19.1` with https://github.com/usememos/memos/commit/00c47a067332d0767075a961cb07ac76b3bea180
